src/configuration/services/mysql.md

```diff
@@ -67,6 +67,8 @@ if (err != nil) {
 }
 {%- endcodetabs %}
 
+{%- language name="Java", type="java", url="https://examples.docs.platform.sh/java/mysql" -%}
```

Comment:
This needs to go up above the `endcodetabs` line, not after.

Reply:
Thank you